website/theme.config.tsx

```diff
@@ -23,35 +21,34 @@ export default defineConfig({
   siteName,
   docsRepositoryBase: 'https://github.com/the-guild-org/the-guild-website/tree/master/website', // base URL for the docs repository
   navbar: {
-    component: (
-      <Header
-        sameSite
-        accentColor="var(--colors-accent)"
-        themeSwitch
-        searchBarProps={{ version: 'v2' }}
-      />
-    ),
+    component: <Header sameSite accentColor="var(--colors-accent)" themeSwitch />,
   },
-  useNextSeoProps() {
-    const { frontMatter, title } = useConfig();
-    const { description, authors, tags, thumbnail, date, updateDate } = frontMatter;
-    const image =
-      thumbnail || `https://og-image.the-guild.dev/?product=GUILD&title=${encodeURI(title)}`;
+  head: function useHead() {
+    const { frontMatter, title: pageTitle } = useConfig();
 
-    return {
-      description: description || `${siteName}: Modern API Platform and Ecosystem that scales`,
-      openGraph: {
-        siteName,
-        images: [{ url: ensureAbsolute(image) }],
-        article: date && {
-          authors: authors && asArray(authors).map(authorId => AUTHORS[authorId]?.name || authorId),
-          publishedTime: new Date(date).toISOString(),
-          modifiedTime: new Date(updateDate || date).toISOString(),
-          tags: tags && asArray(tags),
-        },
```

Comment:
@kamilkisiela do we really need these? I didn't found meta tags for them in `<head/>`